zigbee: Introduce groups shell cmds and extend exitsting cmds to handle groups [KRKNWK-9560] (#7122)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

    The library is no longer directly referenced from the Event Manager.
    Instead, it uses the Event Manager hooks to connect with the manager.

Libraries for Zigbee
--------------------

* :ref:`lib_zigbee_shell`:

> 

Reviewer: What about an entry for the whole group of ``zcl groups`` shell commands?

Author: I don't quite understand what you mean, could you elaborate?

Author: done

>   * Extended zcl cmd shell command to allow sending groupcasts.

Reviewer: ```suggestion
  * Extended ``zcl cmd`` shell command to allow sending groupcasts.
```

>   * Extended zdo shell commands to allow binding to a group addresses.

Reviewer: ```suggestion
  * Extended ``zdo`` shell commands to allow binding to group addresses.
```

>   * Fixed an issue where printing binding table containing group-binding entries results in corrupted output.

Reviewer: ```suggestion
  * Fixed an issue where printing binding table containing group-binding entries would result in corrupted output.
```
